src/core/quicdef.h

```diff
@@ -490,6 +490,11 @@ CXPLAT_STATIC_ASSERT(
 //
 #define QUIC_CONGESTION_CONTROL_ALGORITHM_DEFAULT   QUIC_CONGESTION_CONTROL_ALGORITHM_CUBIC
 
+//
+// The default idle time to change source cid before flush
```

Comment:
How about: "The default idle timeout period after which the source CID is updated before sending again."